indico/modules/users/controllers.py

```diff
@@ -639,3 +639,17 @@ class RHUserSearchInfo(RHProtected):
     def _process(self):
         external_users_available = any(auth.supports_search for auth in multipass.identity_providers.itervalues())
         return jsonify(external_users_available=external_users_available)
+
+
+class RHUserBlock(RHUserBase):
+    def _process_PUT(self):
+        self.user.is_blocked = True
+        logger.info("User %s blocked %s's access", session.user, self.user)
```

Comment:
Can be more concise: `'User %r blocked %r'` (it goes to the logfile, so no need to write verbose sentences)

Reply:
Agree